src/platform/packages/shared/kbn-scout/src/playwright/runner/run_tests.ts

```diff
@@ -134,15 +134,20 @@ export async function runTests(log: ToolingLog, options: RunTestsOptions) {
   const pwConfigPath = options.configPath;
   const pwProject = getPlaywrightProject(options.testTarget, options.mode);
 
-  const pwBinPath = resolve(REPO_ROOT, './node_modules/.bin/playwright');
+  const pwBinPath = resolve(REPO_ROOT, './scripts/playwright');
```

Comment:
@dmlemeshko I've added a single entrypoint that we can use separately - that entrypoint is scripts/playwright. It sets up babel etc, and also passes through the log level flags. The log level flags are then removed in scripts/playwright and converted into an env variable so they don't interfere with Playwright. Let me know if that works for you, otherwise let's hop on a call to figure it out.